### PR TITLE
Develop - HomeView 

### PR DESCRIPTION
**Pull request by manuel-mmr**
HomeView contains the main elements:
- Toolbar to show the name of the app
- SideNav left, that is not showed
- Ui-view to include the LoginView and the ExamView

### CODE REVIEW

File: examsApp/app/home/home.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.toolbar-title{
+    margin-left: 15px;
+}
+.main-layout{
+    width: 40%;
+    margin-left: auto;
+    margin-right: auto;
+}
```
Comment (juanmct) [inline examsApp/app/home/home.scss]: Please, add a newline to help github see the differences

File: examsApp/app/home/home.tpl.jade [hunk 2]
```diff
@@ -1,2 +1,16 @@
...
+        md-toolbar.md-whiteframe-20dp(layout="row", layout-align="space-between center")
+            h5.toolbar-title CogniExams
+            
+        md-content(flex, md-scroll-y)
+            div(layout="column", layout-fill, layout-padding)
+                div(data-ui-view)
+                
+                   
+
+
```
Comment (juanmct) [inline examsApp/app/home/home.tpl.jade]: Same as previous comment

File: examsApp/app/home/home.tpl.jade [hunk 3]
```diff
@@ -1,2 +1,16 @@
-h2 home
-p {{home.ctrlName}}
\ No newline at end of file
+div(layout="row", ng-cloak)
+    md-sidenav.md-whiteframe-20dp.md-sidenav-left(layout="column", md-component-id="left", md-is-locked-open="false")
+        md-toolbar.md-whiteframe-20dp(layout="row", layout-align="space-between center")
+            h5 CogniExams
+            
+    div.relative(layout="column", layout-fill, role="main")
```
Comment (juanmct) [inline examsApp/app/home/home.tpl.jade]: You don't need to use div having the selector in jade

File: examsApp/app/home/home.tpl.jade (hunk 2 above)
Comment (manuel-mmr) [inline examsApp/app/home/home.tpl.jade]: Fixed

File: examsApp/app/login/login.tpl.jade [hunk 4]
```diff
@@ -0,0 +1,12 @@
+md-card.main-layout.md-whiteframe-10dp(layout="column", layout-fill, layout-padding)
+    form(ng-submit="submit(user)" name="login" novalidate)
+        md-input-container.input-style
+            label Username
+            input(ng-model="user.username", required="")
+        md-input-container.input-style
+            label Password
+            input(ng-model="user.password", type="password", required="")
+            
+        div.button-container(layout="row", layout-fill, layout-padding)
```
Comment (juanmct) [inline examsApp/app/login/login.tpl.jade]: div is not needed here as a selector is added

**Comment by juanmct** [comment]
Please, check the unitTests. They fail.

**Comment by manuel-mmr** [comment]
Tests were fixed and they are working correctly now.

**Pull request merged**